README.md

```diff
@@ -1155,6 +1155,7 @@ Update Time, five active automations, webhooks.
 
   * [Alwaysdata](https://www.alwaysdata.com/) — 100 MB free web hosting with support for MySQL, PostgreSQL, CouchDB, MongoDB, PHP, Python, Ruby, Node.js, Elixir, Java, Deno, custom web servers, access via FTP, WebDAV and SSH; mailbox, mailing list and app installer included.
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
+  * [Bohr](https://bohr.io) — Developer-First Deployment and Development Platform that minimizes infrastructure hassle and speed up setup. Code more, manage less.
```

Comment:
please add something like "free for non commercial projects", we can also remove "Code more, manage less." its just noise that doesnt add value, space better used to explain what this actuall does